web/_js/time.js

```diff
@@ -97,41 +97,80 @@ let slider = document.getElementById("timeControlsSlider");
 let tooltip = document.getElementById("timeControlsTooltip")
 let image = document.getElementById("image");
 
-let timeCallback = (a) => {};
-let atlasBackup = [];
+let defaultPeriod = timeConfig.length
+var period = defaultPeriod
+window.period = period
 
 // SETUP
-slider.max = timeConfig.length;
-slider.value = timeConfig.length;
+slider.max = timeConfig.length - 1;
+document.querySelector('#startPeriodField').max = defaultPeriod
+document.querySelector('#endPeriodField').max = defaultPeriod
```

Comment:
Why are you setting `slider` to `defaultPeriod - 1` and both `startPeriodField` and `endPeriodField` to `defaultPeriod`?
This leads to the error, that upon entering the draw mode on an entry without period, the error `Not visible during this period!` is always displayed, as `period` will be 14 in draw.js:297, and the values of both sliders will be 15.

Reply:
Yeah, this is some night owl job. Fixed it.